inst/tests/tests.Rraw

```diff
@@ -1853,10 +1837,10 @@ test(609, chorder(character()), base::order(character()))
 test(610, chorder(""), base::order(""))
 # Extra tests of chorder and chgroup
 x = sample(LETTERS)
-test(610.1, chorder(x), with_c_collate(base::order(x)))
+test(610.1, chorder(x), base::order(x, method="radix"))
 test(610.2, chgroup(x), seq_along(x))
 x = sample(LETTERS,1000,replace=TRUE)
-test(610.3, chorder(x), with_c_collate(base::order(x)))
+test(610.3, chorder(x), base::order(x, method="radix"))
```

Comment:
I would keep the old test. Radix implies C locale, or at least should, but we are not testing exactly the same anymore... Anyway test should be ideally vs generated output and not a function call

Reply:
Thanks... I see your point about the former -- using `method="radix"` is actually a proxy way to go about testing what we want here, explicitly calling out `with_c_collate()` makes the intent clearer & more direct.

I don't agree about the latter point, in some tests we are explicitly trying to match a base equivalent, in the other we don't gain much by having a vector of randomly-ordered integers, function call to base `order()` is clearer.